eo-phi-normalizer/test/eo/phi/metrics.yaml

```diff
@@ -37,7 +37,23 @@ tests:
       ⟧
     }
   metrics:
-    dataless: 5
-    applications: 8
-    formations: 5
-    dispatches: 24
+    programMetrics:
+      dataless: 0
```

Comment:
Can you provide an explanation why this is 0?

Reply:
@fizruk, it's because of the new rules.

From the Example 3.32 I inferred that a lambda asset also has a height zero.
![image](https://github.com/objectionary/normalizer/assets/48378098/d6b91633-e82a-4a9e-949f-126a4a5d0dc6)
![image](https://github.com/objectionary/normalizer/assets/48378098/505f719c-89c7-4fc6-8dcb-e295cc964ee8)

Reply:
I see the following dataless objects:
- `Φ.org`
- `Φ.org.eolang`
- `Φ.org.eolang.prints-itself`
- `Φ.org.eolang.prints-itself-to-console`

Reply:
You probably meant `dataless formations`.

- `Φ.org.eolang` and `Φ.org` have lambda-assets. I asked about lambda-assets in https://github.com/yegor256/phi-paper/issues/15#issuecomment-2008962304
- `Φ.org.eolang.prints-itself` has the height 1 because the minimal height of its attributes is 0.
- `Φ.org.eolang.prints-itself-to-console` has the height 1 because the minimal height of its attributes is 0.

Reply:
I think we can merge and fix the rule when it is finalized upstream.